core/templates/services/context.service.spec.ts

```diff
@@ -213,6 +282,12 @@ describe('Context service', () => {
       () => {
         expect(ecs.canAddOrEditComponents()).toBe(true);
       });
+
+    it('should correctly affirm that page contains reference to skills',
+      () => {
+        expect(ecs.getPageContext()).toBe('story_editor');
+        expect(ecs.canEntityReferToSkills()).toBe(false);
```

Comment:
I don't see a precheck for this. Can we add one please?

Reply:
Ditto: spyOn in beforeEach

Reply:
Hm, I think we need a precheck for this, right?

Reply:
Done!